File: examples-sdk/csharp/027-basic-vilserver-minimal/Main.cs
// 027-basic-vilserver-minimal — C# SDK equivalent
// Compile: vil compile --from csharp --input 027-basic-vilserver-minimal/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("app", 8080);
server.Compile();

File: examples-sdk/zig/027-basic-vilserver-minimal/main.zig
// 027-basic-vilserver-minimal — Zig SDK equivalent
// Compile: vil compile --from zig --input 027-basic-vilserver-minimal/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("app", 8080);
    server.compile();
}
